### ISSUE TITLE
TypeError: must be type, not classobj

### ISSUE THREAD

**Issue by nishio** [icoxfog417/pykintone#1]
On Python 2.7, `app.select()` raises TypeError

```
/usr/local/lib/python2.7/dist-packages/pykintone/result.pyc in __init__(self, response)
     24
     25     def __init__(self, response):
---> 26         super(SelectResult, self).__init__(response)
     27         self.records = []
     28         if self.ok:

TypeError: must be type, not classobj
```

It is because you use old-type class. 
```
>>> class A(): pass
>>> super(A)
TypeError: must be type, not classobj

>>> class B(object): pass
>>> super(B)
<super: None, None>
```

So I think `class Result():` should be `class Result(object):`.

**Comment by icoxfog417** [comment]
Thank you for your reporting. But `pykintone` only supports Python3 now.

However, adding `object` to class declaration is compatible fix. So I'll revise it.

**Comment by icoxfog417** [comment]
It is fixed in `0.1.3`.

I'll support compatible fix for Python2, but don't supports it actively.
